src/index.css

```diff
@@ -11,3 +11,75 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+nav.navbar{
+  background-color: #0b1e42;
+}
+
+.nav-item a{
+  color: white;
+}
+
+.cart-icon{
+  width: 30px;
+  height: 30px;
+}
+
+.cart-count{
+  font-size: 10px;
+  color: black;
+  background: yellow;
+  border-radius: 50%;
+  width: 16px;
+  height: 16px;
+  position: absolute;
+  top: 0;
+  left: -4px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.home{
+  padding-top: 40px;
+}
+
+.product-item{
+  margin: 10px 0;
+}
+
+.product-item .card{
+  padding: 10px 0;
+}
+
+.card-body::before{
+  content:"";
+  display: block;
+  height: 1px;
+  background: #c8c5c5;
+  margin: 10px 0
+}
+
+.product-title{
+  font-weight: bold;
+  font-size: 18px;
+}
+
+.product-price{
+  color: #ab0505
+}
+
+.product-des{
+  font-size: 14px;
+}
+
+button.btn.btn-secondary{
```

Comment:
Do we need to use element selector here?. I think `.btn-secondary {}` is enough as it makes our code cleaner and easier to see